src/pages/civic/index.vue

```diff
@@ -145,14 +145,27 @@ export default {
 
     introVideoVimeoId() {
       switch (this.getLocale) {
-        case 'zh-Hant':
+        case 'zh-Hant': {
+          if (this.isCantonese) return '334865461';
           return '334615825';
-
+        }
         case 'en':
         default:
           return '334616132';
       }
     },
+    isCantonese() {
+      if (!process.client) return false;
+      return (
+        navigator &&
+        ((navigator.language &&
+          navigator.language.toLowerCase().includes('hk')) ||
+          (navigator.languages &&
+            navigator.languages.find(lang =>
```

Comment:
suppose navigator.language is one of navigatior.languages?

Reply:
` In Chrome, navigator.language is the language of the browser UI, and is not guaranteed to be equal to navigator.languages[0].`
not sure if thats good or bad..